concepts/nums-strs/about.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+
+AWK is a very small language and it just has a basic set of data types.
+In this lession we'll go over numbers and strings.
+
+## Numbers
+
+Numbers in AWK are implemented double-precision floating point numbers.
+There is no distinction between integers and floats.
+
+AWK understands octal number constants (starting with `0`) and hexadecimal number constants (starting with `0x` or `0X`).
+
+### Numeric operations
+
+You can use the [standard arithmetic operators][ops-arith] `+`, `-`, `*`, `/`, `%`.
+Both `^` and `**` can be used for exponentiation, but because `^` is specified by POSIX it is preferred.
+There are some [builtin numeric functions][funcs-numeric], including trigonometric functions, random number generation, and `int(x)` to truncate a number to the nearest integer between _x_ and zero.
+
+## Strings
+
+AWK strings are composed of ASCII bytes.
+Specify a string constant using double quotes.
+
+### String operations
+
+There is only one string operations: concatenation.
+Concatenation does not have a special operator;
+simply placing two strings next to each other concatenates them:
+
+```awk
+BEGIN {
+    hw = "Hello" "World"
+    print hw
```

Comment:
Are we assuming functions use the `func()` syntax to call them? Assuming students know the `()` are optional at this point?

Reply:
`print` is not a function, it's a statement. https://www.gnu.org/software/gawk/manual/html_node/Print.html

(`printf` too, but `sprintf` is a function).

I'll have to add a note to avoid ambiguity.